params/config/sim_config.yaml

```diff
@@ -9,3 +9,4 @@ flow_size_shape: 0.001             # default: 0.001
 # if deterministic = True, the simulator reinterprets and uses inter_arrival_mean and flow_size_shape as fixed
 # deterministic values rather than means of a random distribution
 deterministic: True                # default: False
+training: False                    # default: False  
```

Comment:
Why is this a config option?

Reply:
I forgot about this. It is obsolete and no longer used.